tests/test_host_check.py

```diff
@@ -1653,6 +1578,8 @@ def test_insufficient_memory(self, capsys):
         assert output == textwrap.dedent(
             """\
             FAIL -- Hugepages
+                    2MiB hugepages are available, but only 1GiB hugepages are
+                    supported for XRd deployment use cases.
                     Only 1.0GiB of hugepage memory available, but XRd
                     requires at least 3GiB.
             """
```

Comment:
Should we have separate 'insufficient memory' tests for the cases of 2M and 1G being used?